### ISSUE TITLE
Stuck while Running Laravel Project After Installing Package

### ISSUE THREAD

**Issue by tinebasilio** [harishdurga/laravel-quiz#48]
Hello there! I'm a beginner with Laravel and I recently forked the repository to work on a project. I successfully cloned the repository. However, I am currently stuck right after running the `composer require harishdurga/laravel-quiz` command. attempted to start the Laravel development server with `php artisan serve`, but there is no existing artisan file in the root directory. I also tried to run `composer install` but it's not working.

As a beginner, I’m unsure about the next steps required to properly set up and run the project locally. Are there additional setup steps that might be needed to get the project running? Thank you so much in advance!

**Comment by harishdurga** [comment]
You had run `composer require harishdurga/laravel-quiz` command inside a Laravel project right?

**Comment by tinebasilio** [comment]
Yes, I ran the` composer require harishdurga/laravel-quiz` command inside the Laravel project. I’ve attached a screenshot of the terminal output for your reference.

![Screenshot 2024-09-01 090240](https://github.com/user-attachments/assets/011f7461-b2d6-457c-8970-56c5f44b0625)

**Comment by devope** [comment]
@tinebasilio it is not a package related problem, you have an error with `artisan` command.
Check your system configuration.
